libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -290,7 +290,11 @@ void GCS::update_sensor_status_flags()
     if (airspeed && airspeed->enabled()) {
         control_sensors_present |= MAV_SYS_STATUS_SENSOR_DIFFERENTIAL_PRESSURE;
         const bool use = airspeed->use();
+#if AP_AHRS_ENABLED
         const bool enabled = AP::ahrs().airspeed_sensor_enabled();
+#else
+        const bool enabled = true;
```

Comment:
this doesn't seem right. We should check AP_Airspeed and if that fails, should be false

Reply:
in fact, here's the code behind that ahrs function call:
```
    static bool airspeed_sensor_enabled(void) {
    #if AP_AIRSPEED_ENABLED
        const AP_Airspeed *_airspeed = AP::airspeed();
        return _airspeed != nullptr && _airspeed->use() && _airspeed->healthy();
    #else
        return false;
    #endif
    }

```

Reply:
AFAICS we onlyt need to choose true or false here?

Are you suggesting false?

Reply:
well, defaulting to true is definitely the wrong "guess" to make. So yeah, I think FALSE is better here. In a periph, even the concept of "use" is pretty sketchy.

Reply:
OK, so you have a periph which also happens to do mavlink.

It's sending airspeed messages via CAN.

AHRS is not compiled in.

You're looking at the SYS_STATUS message.  What should we say in the three bits?

I think:
Present: yep, obviously
Enabled: yes, because we're using it to send messages
Healthy: the result of the healthy() call onto the airspeed library

So I think I should change this "enabled" to `_airspeed != nullptr && _airspeed->use()` - do you agree?

Reply:
.... I've changed this to be as @magicrub suggested, but without the healthy check.

The method on AHRS - `airspeed_sensor_enabled` - needs to die.